src/app/features/app.component.html

```diff
@@ -1,7 +1,8 @@
 <app-header></app-header>
 
-<app-social-links fxHide.xs="true"
-                  fxHide.sm="true"></app-social-links>
+<app-social-links fxHide.xs="true" fxHide.sm="true"></app-social-links>
```

Comment:
This is just organization, but is better if we always put each properties in one line. In this case should be like this:

```
<app-social-links fxHide.xs="true"
                  fxHide.sm="true"
```

Reply:
When I save the code it is automatically stylized like that